halo2_proofs/src/poly/kzg/commitment.rs

```diff
@@ -132,6 +132,27 @@ where
         }
     }
 
+    /// Initializes parameters for the curve through existing parameters
+    /// k, n, g, g_lagrange, g2, s_g2
+    pub fn from_params(
+        &self,
+        k: u32,
+        n: u64,
```

Comment:
This doesn't need to be provided if we have `k` as `n = 1<<k`.
Could you remove it?